src/MainWindow.vala

```diff
@@ -274,14 +274,22 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
             maximize ();
         }
 
+        show_all ();
```

Comment:
IMHO, calling `show_all()` inside `construct()` is a bad pattern, since not all properties is actually with the right values.

Doing it for the box is fine though, since in Gtk4 child widgets are shown by default, and `present()`/`present_with_time()` will call `show()` internally.

```suggestion
        box.show_all ();
```